ppr-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "1.1.4",
+  "version": "1.1.5",
```

Comment:
* The change differential is mostly package-lock items, don't get scared off haha
* Hide white space changes for easier reviewing.